hack/generate-swagger-api.sh

```diff
@@ -26,8 +26,8 @@ swagger generate model -f api/swagger.yaml \
 
 swagger generate model -f api/swagger.yaml \
 	-t api -m types/image --skip-validator -C api/swagger-gen.yaml \
-	-n ImageDeleteResponseItem \
-	-n ImageSummary
+	-n ImageDeleteResponseItem
+#-n ImageSummary TODO: Restore when go-swagger is updated
```

Comment:
Can you add a link to the comment you left on this PR? This may be one of those cases where we don't recall "what specific feature are we waiting for?"

Reply:
Ah, good idea, done

Reply:
Oh! It's the line below; I thought you forgot to push 😂 thanks!